jobs/e2e_node/perf-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
+  cosbeta-resource1:
+    image: cos-beta-81-12871-44-0
+    project: cos-cloud
+    machine: n1-standard-16
```

Comment:
can see from https://github.com/kubernetes/kubernetes/pull/65251 that it is supposed to be running on this machine type.